server/views/pages/reviewer/view-statements.html

```diff
@@ -30,42 +31,72 @@ <h1 class="govuk-heading-xl">{{ pageTitle }}</h1>
       <table class="govuk-table" data-qa="statements">
         <thead class="govuk-table__head">
           <tr class="govuk-table__row">
-            <th class="govuk-table__header" scope="col">Staff member involved</th>
+            <th class="govuk-table__header" scope="col">Staff members involved</th>
+            <th class="govuk-table__header" scope="col"></th>
             <th class="govuk-table__header" scope="col"></th>
           </tr>
         </thead>
 
         <tbody class="govuk-table__body">
-            {% for statement in data.statements %}
-            <tr class="govuk-table__row">
-              <td class="govuk-table__cell">{{statement.name }}</td>
-              <td class="govuk-table__cell">
-                {% if statement.isSubmitted %}
+          {% for statement in data.statements %}
```

Comment:
New OVERDUE and REMOVAL REQUEST badges. 
If both conditions are true, then only display the REMOVAL REQUEST